app/services/location_lookup_service.rb

```diff
@@ -59,4 +59,10 @@ def response
                     }
                   end
   end
+
+  def mapit
+    return GdsApi.mapit unless Rails.env.development?
+
+    GdsApi::Mapit.new("https://mapit.mysociety.org/")
```

Comment:
Is it possible to do this on a global level? ie. inside of the GDS API adapter?

Reply:
I kept it like that so it would still throw the same GdsApi errors, i.e. GdsApi::HTTPNotFound and GdsApi::HTTPClientError, otherwise [that bit of the code](https://github.com/alphagov/collections/blob/master/app/services/location_lookup_service.rb#L53) will stop working.

Reply:
Correct me if I'm wrong but wouldn't it still keep these errors as we'll be instantiating with the same class?

Anyways, I think, for now, we can localise the change until we have something better.

Reply:
Oh! sorry! I mis-read! You meant, put this inside of the gem! 🤦 Yes, it should be possible to do that, but I only envisioned this as a stop-gap. There's a bigger problem around how do we access Mapit data locally.